common/landed_titles/wc_titular_titles.txt

```diff
@@ -1418,7 +1418,7 @@ e_herald_cthun = {
 	color = { 243 180 17 }
 	color2={ 255 255 255 }
 	
-	capital = 2956 # c_ahnqiraj
+	#capital = 2956 # c_ahnqiraj
```

Comment:
Title must always have capital. In case, title becomes landless, its ruler is located in capital.

Reply:
it works fine currently, what is the problem?

Reply:
They have no location. So, for example, they always return false when you try to check it. It may result in many bugs, so you need to tell why you changed it at least.
Note: Yogg-Saron Empire in my version has capital, C'thun - not.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/73769605-6563e980-4794-11ea-9131-66e4aefb0aa2.png)

</details>